StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheet+API.swift

```diff
@@ -529,6 +535,10 @@ extension PaymentSheet {
                     case .failure(let error as NSError):
                         STPAnalyticsClient.sharedClient.logLinkSignupFailure(error: error)
                         // Attempt to confirm directly with params as a fallback.
+                        intentConfirmParams.setAllowRedisplay(
+                            mobilePaymentElementFeatures: elementsSession.customerSessionMobilePaymentElementFeatures,
+                            isSettingUp: intent.isSetupFutureUsageSet(for: intentConfirmParams.paymentMethodParams.type)
+                        )
```

Comment:
If we were unable to create the Link account, we should still respect the merchant's request to allow redisplay on this payment method.